tests/js/spec/views/team/teamNotificationSettings.spec.jsx

```diff
@@ -0,0 +1,105 @@
+import {mountWithTheme} from 'sentry-test/enzyme';
+import {initializeOrg} from 'sentry-test/initializeOrg';
+
+import TeamNotificationSettings from 'app/views/settings/organizationTeams/teamNotifications';
+
+const EXTERNAL_NAME = 'marcos';
+const EXAMPLE_EXTERNAL_TEAM = {
+  externalName: EXTERNAL_NAME,
+  id: '1',
+  integrationId: '1',
+  provider: 'slack',
+};
+const EXAMPLE_INTEGRATION = {
+  id: '1',
+  provider: {
+    key: 'slack',
+  },
+};
+
+describe('TeamNotificationSettings', () => {
+  let team;
+
+  beforeEach(() => {
+    MockApiClient.clearMockResponses();
+    team = TestStubs.Team();
+  });
+
+  it('should render empty message when there are no integrations', () => {
+    const {organization, routerContext} = initializeOrg();
+
+    MockApiClient.addMockResponse({
+      url: `/teams/${organization.slug}/${team.slug}/`,
+      body: {
+        externalTeams: [EXAMPLE_EXTERNAL_TEAM],
+      },
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/integrations/`,
+      body: [],
+    });
+
+    const wrapper = mountWithTheme(
+      <TeamNotificationSettings team={team} organization={organization} />,
+      routerContext
+    );
+
+    const emptyMessage = wrapper.find('Panel div[data-test-id="empty-message"]');
+    expect(emptyMessage).toHaveLength(1);
+    expect(emptyMessage.text()).toBe(
+      'No Notification Integrations have been installed yet.'
+    );
+  });
+
+  it('should render empty message when there are no externalTeams', () => {
+    const {organization, routerContext} = initializeOrg();
+
+    MockApiClient.addMockResponse({
+      url: `/teams/${organization.slug}/${team.slug}/`,
+      body: {
+        externalTeams: [],
+      },
+    });
+
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/integrations/`,
+      body: [EXAMPLE_INTEGRATION],
+    });
+
+    const wrapper = mountWithTheme(
+      <TeamNotificationSettings team={team} organization={organization} />,
+      routerContext
+    );
+
+    const emptyMessage = wrapper.find('Panel div[data-test-id="empty-message"]');
+    expect(emptyMessage).toHaveLength(1);
+    expect(emptyMessage.find('div div div').first().text()).toBe(
+      'No External Teams have been linked yet.'
```

Comment:
I know this is a bit of a tangent and not the point of this PR, but is this the user facing message? I doubt a user would know what an external team is

Reply:
Yes this is rendered to the page. I'll just quickly change it to "teams".